src/app.ts

```diff
@@ -0,0 +1,32 @@
+import express from "express";
+import mongoose from "mongoose";
+import cookieParser from "cookie-parser";
+import dotenv from "dotenv";
+import userRouter from "./routes/user.routes";
+import recipeRouter from "./routes/recipe.routes";
+
+dotenv.config();
+const app = express();
+
+app.set("port", process.env.PORT || 3000);
+app.use(express.json());
+app.use(cookieParser());
+
+mongoose.connect(
```

Comment:
You can export mongoose config to separate file, e.g. `./config/db_config.ts`, and perhaps declare some init() function there